packages/js/product-editor/src/components/content-preview/content-preview.tsx

```
@@ -80,10 +80,12 @@ export function ContentPreview( { content }: ContentPreviewProps ) {
 					</style>
 					<div
 						className="woocommerce-content-preview__content"
-						dangerouslySetInnerHTML={ sanitizeHTML( content, {
-							tags: CONTENT_TAGS,
-							attr: CONTENT_ATTR,
-						} ) }
+						dangerouslySetInnerHTML={ {
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)